<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherManger</class>
 <widget class="QWidget" name="teacherManger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>teacherManger</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成绩管理</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>761</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>111</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
